README.CentOS7.md

```diff
@@ -0,0 +1,157 @@
+```
+
+## PHP Composer
+```
+sudo curl -sS https://getcomposer.org/installer | php
+sudo mv composer.phar /usr/local/bin/composer
+```
+# Get the Code
+
+Download the latest release from the [releases page](https://github.com/aces/Loris/releases) to the home directory (~/), unzip it, and copy the contents to your project directory, `/var/www/loris` (we recommend naming your project directory `loris`, although you can use a different naming convention if you prefer). 
+```
+wget https://github.com/aces/Loris/archive/v$VERSION.tar.gz
+tar -zxf Loris-%VERSION%.tar.gz
+cp -r Loris-%VERSION%/ /var/www/loris
+```
+
+Alternatively the latest development branch can be obtained by forking the [LORIS repository](http://github.com/aces/Loris) for development purposes. We do not support unstable dev branches. 
+
+# Configure Apache
+
+A sample apache configuration file is in `docs/config/apache2-site`. 
+The install script will ask if you want to automatically create/install apache config files.
+It is recommended to perform this step **manually** --: copy our sample file to the apache configuration directory (`/etc/httpd/conf.d/`) and add the `.conf` file extension:
+
+```bash
+cd /var/www/loris
+cp docs/config/apache-site /etc/httpd/conf.d/apache-site.conf
+```
+
+Customize and Verify your settings: 
+* Paths and settings in `/etc/httpd/conf.d/apache-site.conf` should be populated appropriately for your server. Replace placeholders such as `%LORISROOT%` with `/var/www/loris`, `%PROJECTNAME%` with `loris`, `%LOGDIRECTORY%` with `/var/log/httpd/loris-error.log` 
+ * DocumentRoot should point to `/var/www/loris/htdocs`
+ * The `smarty/templates_c/` directory must be writable by Apache (e.g. by running: `sudo chgrp -R httpd /var/www/loris/smarty/templates_c` and `sudo chmod 775 /var/www/loris/smarty/templates_c`).
+
+Finally, restart apache:
+```
+sudo systemctl restart httpd
+sudo systemctl status httpd
+```
+# Install LORIS
+
+For the purpose of following LORIS conventions and easy understanding of all LORIS documentation, we recommend the following account names: 
+
+* `lorisadmin` unix user with sudo permission who will setup and manage the LORIS back-end
+* `lorisuser` MySQL database user with limited (insert, delete, update, select...) permissions, for database transactions requested by the Loris front-end
+* `admin` default front-end username for LORIS administrator account (login via browser)
+
+## Run the LORIS install script to set up directories
+
+```
+cd /var/www/loris/tools
+./install.sh
+```
+Run the makefile (use `make dev` if you are setting up a development sandbox)
+```
+cd /var/www/loris
+make
+```
+## Install your database
+Open your browser and go to: `http://%IPADDRESS%/installdb.php`
+
+(%IPADDRESS% will likely be the IP address of the VM you are ssh'ed into)
+
+The web page will prompt you for the following information: 
+
+ * `Server Hostname` localhost if your database if hosted on your VM or the IP address of your database server
```

Comment:
```suggestion
 * `Server Hostname` localhost if your database is hosted on your VM or the IP address of your database server
```

Reply:
Thanks @johnsaigle -  I just fixed this typo in [commit b2080db](https://github.com/aces/Loris/pull/5203/commits/b2080dbb39694f974074dd834b271df6b1be1c72) - could you verify/resolve?